### PR TITLE
Update footer for 2020

### PR DESCRIPTION
**Pull request by justjavac**
Happy new year 2020

### CODE REVIEW

**Comment by jaeh** [comment]
shouldn't this be 2019 - 2020?

these disclaimers are supposed to be used in copyright lawsuits for proof of provenance,
for example if somebody else registers hyperapp as a trademark and tries to get npm to change the name. thats why they should signify the date of copyright creation. the current year can be added optionally.

not that those disclaimers have any legal weight, but that's another story ;)

happy 2020 to you too :)

File: docs/index.html [hunk 1]
```diff
@@ -128,7 +128,7 @@ <h1>the tiny framework <br />for building web interfaces</h1>
         <br />
         Released under the MIT License
         <br />
-        Copyright © 2019
+        Copyright © 2020
```
Comment (frenzzy) [inline docs/index.html]:
```suggestion
        Copyright © 2019-<script>document.write(new Date().getFullYear())</script>
```

**Comment by jorgebucaran** [comment]
@lotteo Maybe this should be part of an app? 

`document.write` 
☝️😏

**Comment by jorgebucaran** [comment]
@justjavac @loteoo Maybe this should be part of an app? 

`document.write` 
☝️😏

**Comment by justjavac** [comment]
done

**Comment by jorgebucaran** [comment]
Ah, that's a simpler declarative solution! 😄

**Pull request merged**